src/test/rpc/Book_test.cpp

```diff
@@ -800,6 +800,460 @@ class Book_test : public beast::unit_test::suite
             books)[jss::status] == "success");
     }
 
+    void testTrackOffers(void)
+    {
```

Comment:
@seelabs Is the suggestion to call `testcase("name of test")` at the beginning of each test? I've seen that in other tests but not uniformly across the codebase.